modules/test/playwright/tests/headless-builder-web/get.spec.ts

```diff
@@ -128,13 +177,26 @@ test('can see available path parameter properties of a singleElement endpoint',
 
 	await headlessBuilderPage.goto();
 	await headlessBuilderPage.deleteApplication('My-app');
+	await objectAdminRestClient.objectDefinition.deleteObjectDefinition({
```

Comment:
This can be done by using the data array of the apiHelpers:

https://github.com/liferay/liferay-portal/blob/fd0654eb708986381fe80455e36b0df607eb4171/modules/test/playwright/tests/roles-admin-web/objectsRolePermissions.spec.ts#L119

That will be autoimatically handled in the tearDown

Reply:
yes, will do, thanks! :D